SofaKernel/modules/SofaBaseTopology/src/SofaBaseTopology/EdgeSetGeometryAlgorithms.h

```diff
@@ -157,6 +157,11 @@ class EdgeSetGeometryAlgorithms : public PointSetGeometryAlgorithms<DataTypes>
     /** return a pointer to the container of cubature points */
     NumericalIntegrationDescriptor<Real,1> &getEdgeNumericalIntegrationDescriptor();
 
+    bool computeEdgeSegmentIntersection(EdgeID edgeID,
```

Comment:
Not the point of the PR as it is also done in the existing code but it would be better to use Real instead of double.